### ISSUE TITLE
how to use migrator?

### ISSUE THREAD

**Issue by xiaoyulei** [docker/v1.10-migrator#18]
https://github.com/docker/docker/wiki/Engine-v1.10.0-content-addressability-migration
From this wiki, it says "All your current images, tags and containers are automatically migrated to the new foundation the first time you start updated Docker daemon."

But how to operator it? I execute v1.10-migrator , it report fail.

./v1.10-migrator 
FATA[0000] no storage driver directory was found at /var/lib/docker

**Comment by thaJeztah** [comment]
Error seems to indicate that `/var/lib/docker` is empty, or possibly you need to run as a privileged user (or with `sudo`). Is docker running with a different graph-directory than `/var/lib/docker`?

**Comment by xiaoyulei** [comment]
@thaJeztah 
`/var/lib/docker` is not empty. I do not change the default directory,all is default configuration. I try to use sudo, but it is the same.

**Comment by thaJeztah** [comment]
@YuleiXiao what version do you have installed currently (i.e., before 1.10)

**Comment by xiaoyulei** [comment]
@thaJeztah 1.9

**Comment by maxclaus** [comment]
That problem occurs because is running the command without `sudo`. At least it was the reason for me.

**Comment by timchenxiaoyu** [comment]
migrator can migrate the container from one host to another?

**Comment by thaJeztah** [comment]
> migrator can migrate the container from one host to another?

@timchenxiaoyu no, read the link in first comment here, and the readme of this repository

**Comment by timchenxiaoyu** [comment]
i see, it use to upgrade docker from 1.9 to 1.10 over.
